### PR TITLE
Explain error structs in try-rescue

### PR DESCRIPTION
**Pull request by angelikatyborska**
Closes https://github.com/exercism/elixir/issues/927

@petemcfarlane do you think that sort of addition would be helpful? (Note: `about.md` is shown only after completing a concept exercise)

### CODE REVIEW

File: concepts/try-rescue/introduction.md [hunk 1]
```diff
@@ -22,3 +22,12 @@ Let's examine this construct:
...
     - the instructions to be executed if the error matches.
+
+## Error structs
+
+Errors (sometimes also called "exceptions") that you rescue this way are structs.
+Rescuing errors in Elixir is done very rarely.
+Usually the rescued error is logged or sent to an external monitoring service, and then reraised.
+This means we usually don't care about the internal structure of the specific error struct.
+
+In the [Exceptions concept](/tracks/elixir/concepts/exceptions) you will learn more about error structs, including how to define your own custom error.
```
Comment (petemcfarlane) [inline concepts/try-rescue/introduction.md:33]: I think this bit is good 👍

**Review by petemcfarlane — APPROVED** [review]
Yeah I understand that you don't want to swap the order. Adding this info does help I think, thanks 🙂

File: concepts/try-rescue/about.md [hunk 2]
```diff
@@ -22,6 +22,26 @@ end
   - on the right side:
     - the instructions to be executed if the error matches.
 
+## Error structs
+
+Errors (sometimes also called "exceptions") that you rescue this way are structs.
+Different error structs have different keys.
+Under the ["exceptions" section][argument-error] in the standard library you can find a list of all possible errors.
```
[the remark this answers is not in the document]
Comment (angelikatyborska) [reply concepts/try-rescue/about.md:29]: Right!

**Review by jiegillet — APPROVED** [review]
If @petemcfarlane is happy with this, so am I!

**Pull request merged**